tests/integration/test_data/pip_with_deps/output.json

```diff
@@ -0,0 +1,48 @@
+{
+    "packages": [
+        {
+            "type": "pip",
+            "path": ".",
+            "name": "cachito-pip-with-deps",
+            "version": "1.0.0",
+            "dependencies": [
+                {
+                    "type": "pip",
+                    "name": "aiowsgi",
+                    "version": "0.7",
+                    "dev": false
+                },
+                {
+                    "type": "pip",
+                    "name": "appr",
+                    "version": "git+https://github.com/quay/appr.git@58c88e4952e95935c0dd72d4a24b0c44f2249f5b",
+                    "dev": false
+                },
+                {
+                    "type": "pip",
+                    "name": "appr",
+                    "version": "https://github.com/quay/appr/archive/37ff9a487a54ad41b59855ecd76ee092fe206a84.zip#egg=appr&cachito_hash=sha256:ee6a0a38bed8cff46a562ed3620bc453141a02262ab0c8dd055824af2829ee5c",
+                    "dev": false
+                }
+            ]
+        }
+    ],
+    "environment_variables": [
+        {
+            "name": "PIP_FIND_LINKS",
+            "value": "deps/pip",
+            "kind": "path"
+        },
+        {
+            "name": "PIP_NO_INDEX",
+            "value": "true",
+            "kind": "literal"
+        }
+    ],
+    "project_files": [
+        {
+            "abspath": "${absolute_path}/pip_with_deps-source/requirements.txt",
```

Comment:
Minor nitpick: IMO the `absolute_path` should the path to the source directory, not to the parent of the source directory.

And while we're at it, maybe rename it to `test_source_dir` or something like that? The goal would it be to make it clear that

a) this variable is for integration tests, cachi2 doesn't actually put it there
b) the absolute path points to the source directory

Reply:
Or it could be `testcase_tmpdir`, in which case it would be pretty clear what it is and why it points to the parent of the source directory